doc/examples/animations/lib/main.dart

```diff
@@ -32,7 +43,7 @@ class MyGame extends BaseGame with TapDetector {
     final animationComponent = SpriteAnimationComponent.sequenced(
       291,
       178,
-      'creature.png',
+      creature,
```

Comment:
if we use the image twice though, we can use the syntax you used. or rely on flame.images.load caching stuff

Reply:
Like a mentioned before I have a nice idea for this, will implant it and will let you know